### DISCUSSION TITLE
Front matter for "on wake" as well as "startup"?

### DISCUSSION THREAD

**Discussion by RosemaryOrchard** [ttscoff/bunch-docs#226] [Ideas]
I was debugging a Bunch earlier and realised the answer to "why doesn't this run?" is very simple—I've not scheduled it. I put "startup" in the frontmatter, and apparently in my head I assumed it was analogue to "on machine wake".

Is there a way to schedule a Bunch for "when my machine wakes up"? Or would it be best to schedule it for regular times and then take advantage of the catch-up feature?

**Comment by ttscoff** [comment]
"startup: true" means that the bunch will open whenever Bunch (the app) 
launches, which would typically be first login. Bunch doesn't do any 
sleep/wake polling that would allow it to run every time your machine 
wakes. There's an AppleScript screensaver that can trigger a script on 
wake (or at least there was last time I looked), and Keyboard Maestro 
has triggers for wake and unlock that could call a Bunch URL. Would 
either of those solutions help?

**Comment by RosemaryOrchard** [comment]
> Keyboard Maestro has triggers for wake and unlock that could call a Bunch URL

That was my plan as a backup if Bunch didn't have a way to do that! I'll just make sure to document in my Bunch that I've done that because otherwise I will inevitably be back here confused at some point in the future 😂
